editor/editor_inspector.cpp

```diff
@@ -2503,6 +2503,11 @@ void EditorInspector::update_tree() {
 				continue;
 			}
 
+			// Hide the "MultiNodeEdit" category for MultiNodeEdit.
+			if (Object::cast_to<MultiNodeEdit>(object) && p.name == "MultiNodeEdit") {
```

Comment:
The RefCounted problem might be solved by adding a `||` for `p.name` here.

Reply:
The other categories should still show for MultiNodeEdit though. The only categories that need to be hidden are MultiNodeEdit and RefCounted.

A better way to fix that would be to call `_validate_property()` for properties with `PROPERTY_HINT_CATEGORY`, but I couldn't figure out how to do that.